samples/04_gis_analysts_data_scientists/Training a Deep Learning model, for using it in 3D basemap Solutions..ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>In <code>fit( )</code>, the argument 'epochs' controls the maximum number of epochs for which the model will be trained. A maximum epoch of one means the dataset will be passed forward and backward through the neural network one time. <s>And</s> If 'early_stopping' is True, then the model training will stop when the model is no longer improving, regardless of the 'epochs' parameter value specified.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
corrected accordingly, Thanks.